<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StreamToolBox</class>
 <widget class="QToolBox" name="StreamToolBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>476</height>
   </rect>
  </property>
  <property name="currentIndex">
   <number>8</number>
  </property>
  <widget class="QWidget" name="pageOverview">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>388</width>
     <height>58</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <attribute name="label">
    <string>Overview</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <widget class="QCheckBox" name="skipDownloadCheckBox">
      <property name="text">
       <string>Do not download the video, but write all other related files</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="markWatchedCheckBox">
      <property name="text">
       <string>Mark watched (only for Youtube)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="pageSubtitles">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>405</width>
     <height>115</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <attribute name="label">
    <string>Subtitles</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_9">
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="4" column="0">
       <widget class="QCheckBox" name="subtitleConvCheckBox">
        <property name="text">
         <string>Convert to format:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="subtitleExtLabel">
        <property name="text">
         <string>Preferred format:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="subtitleExtComboBox"/>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="subtitleCheckBox">
        <property name="text">
         <string>Download subtitles:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="subtitleConvComboBox"/>
      </item>
      <item row="2" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QComboBox" name="subtitleLangComboBox"/>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QCheckBox" name="subtitleHideAutoCheckBox">
        <property name="text">
         <string>Hide auto-generated subtitles</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="pageChapters">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>405</width>
     <height>103</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <attribute name="label">
    <string>Chapters</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 85, 0);</string>
      </property>
      <property name="text">
       <string notr="true">(Work in progress :)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="chapterSplitCheckBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Split video into multiple files based on internal chapters</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_14">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Remove chapters whose title matches the given regular expression:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="placeholderText">
       <string>Ex: &quot;*10:15-15:00&quot;, or &quot;intro&quot;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="pageThumbnails">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>405</width>
     <height>105</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <attribute name="label">
    <string>Thumbnails</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 85, 0);</string>
      </property>
      <property name="text">
       <string notr="true">(Work in progress :)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="thumbnailCheckBox">
      <property name="text">
       <string>Download thumbnail (default image)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="allThumbnailsCheckBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Download all formats of the thumbnail image</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="checkBox_4">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Convert the thumbnails to format:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string notr="true">jpg</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">png</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="pageComments">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>405</width>
     <height>82</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <attribute name="label">
    <string>Comments</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 85, 0);</string>
      </property>
      <property name="text">
       <string notr="true">(Work in progress :)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="commentCheckBox">
      <property name="text">
       <string>Download comments (in the .info.json)</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="commentSortLabel">
        <property name="text">
         <string>Sort by:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="commentSortComboBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>Newest first</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Top comments</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="commentLimitCheckBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Limit number of comments:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="commentLimitSpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="pageOtherMedia">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>388</width>
     <height>81</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <attribute name="label">
    <string>Other Media</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_10">
    <item>
     <widget class="QCheckBox" name="otherDescriptionCheckBox">
      <property name="text">
       <string>Download description as a .description file</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="otherMetadataCheckBox">
      <property name="text">
       <string>Download metadata as a .info.json file</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="otherShortcutCheckBox">
      <property name="text">
       <string>Create internet shortcut</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="pageProcessing">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>388</width>
     <height>482</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <attribute name="label">
    <string>Pre/Post-Processing</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 85, 0);</string>
      </property>
      <property name="text">
       <string notr="true">(Work in progress :)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="commandCroupBox">
      <property name="title">
       <string>Command</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <item>
        <widget class="QCheckBox" name="checkBox_2">
         <property name="text">
          <string>Execute the command before the actual download:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_3">
         <property name="text">
          <string>Execute the command on the file after downloading and post-processing:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_2"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="postProcessGroupBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>Post-Processing</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QCheckBox" name="checkBox_9">
         <property name="text">
          <string>Remux the video into another container if necessary :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_3">
         <property name="placeholderText">
          <string>Ex: &quot;aac&gt;m4a/mov&gt;mp4/mkv&quot;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Ex: &quot;aac&gt;m4a/mov&gt;mp4/mkv&quot; will remux aac to m4a, mov to mp4 and anything else to mkv.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_8">
         <property name="text">
          <string>Re-encode the video into another format if re-encoding is necessary:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_4">
         <property name="placeholderText">
          <string>Ex: &quot;aac&gt;m4a/mov&gt;mp4/mkv&quot;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Rem: The syntax and supported formats are the same as &quot;Remux&quot;</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_6">
         <property name="text">
          <string>Keep the intermediate video file on disk after post-processing</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_7">
         <property name="text">
          <string>Embed subtitles in the video (only for mp4, webm and mkv videos)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_10">
         <property name="text">
          <string>Embed thumbnail in the video as cover art</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_11">
         <property name="text">
          <string>Embed metadata in the video file</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_12">
         <property name="text">
          <string>Embed chapter markers in the video file</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_13">
         <property name="text">
          <string>Write metadata to the video file's xattrs (using dublin core and xdg standards)</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="pageSponsorBlock">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>405</width>
     <height>220</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <attribute name="label">
    <string>SponsorBlock</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 85, 0);</string>
      </property>
      <property name="text">
       <string notr="true">(Work in progress :)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_15">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Remove segments in SponsorBlock categories from the video file:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item row="3" column="1">
       <widget class="QCheckBox" name="checkBox_21">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Unpaid/Self Promotion</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="checkBox_22">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Interaction Reminder</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="checkBox_16">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Sponsor</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBox" name="checkBox_18">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Preview/Recap</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QCheckBox" name="checkBox_19">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Non-Music Section</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="checkBox_20">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Endcards/Credits</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="checkBox_17">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Intermission/Intro Animation</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="lastPage">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>405</width>
     <height>233</height>
    </rect>
   </property>
   <attribute name="label">
    <string notr="true">&lt;keep last page blank&gt;</string>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
